src/phoenix/db/bulk_inserter.py

```diff
@@ -0,0 +1,177 @@
+import asyncio
```

Comment:
nit: the file location of this feels off if it's specific to spans

Reply:
it could be used for bulk inserting evals too. it'll just need second queue